### ISSUE TITLE
[FEATURE] Improving nav a

### ISSUE THREAD

**Issue by YadlaMani** [krishnaacharyaa/wanderlust#379]
### Description

It would be better if we improve the nav bar and add more details so it increases overall info in the website

### Use Case

The user can go to github repo from navbar

### Proposed Solution

![image](https://github.com/krishnaacharyaa/wanderlust/assets/133841094/d6fa2ac0-48a8-4535-bc25-eb6155a77ac0)

### Attachments

![image](https://github.com/krishnaacharyaa/wanderlust/assets/133841094/5401b185-ed61-4ab8-b998-91c13c6315b3)

**Comment by krishnaacharyaa** [comment]
What exactly would you wanna add @YadlaMani ?

**Comment by YadlaMani** [comment]
@krishnaacharyaa adding git hub link and your name to the footer

**Comment by krishnaacharyaa** [comment]
@YadlaMani no need of my name you can add something like 
<img width="612" alt="image" src="https://github.com/krishnaacharyaa/wanderlust/assets/116620586/6e832e3f-097c-4ddb-bae0-3c62275776de">

**Comment by YadlaMani** [comment]
@krishnaacharyaa okay will do that assign me the issue

**Comment by krishnaacharyaa** [comment]
@EshaalakshmiDS, can you work on this? kindly update this to have proper title and then see if you can do this

**Comment by EshaalakshmiDS** [comment]
@krishnaacharyaa Yes I can work on this, but I am not able to update this to have proper title, can u help with that. 
I have deleted the duplicate issue. Can u please assign this to me under GSSOC'24 label. Thank you
